scripts/Makefile.build

```diff
@@ -277,7 +277,7 @@ $(obj)/%.lst: $(src)/%.c FORCE
 # Compile Rust sources (.rs)
 # ---------------------------------------------------------------------------
 
-rust_allowed_features := core_ffi_c
+rust_allowed_features := core_ffi_c,explicit_generic_args_with_impl_trait
```

Comment:
@ojeda We probably want a rustc update before sending this PR (this feature is stablised in 1.63)

Reply:
Good eye! We were talking about this one yesterday -- I think it is fine, Benno wants to send v1 within a couple of days, and we can land this and the upgrade in parallel. I can take care of removing it when applying his patch to avoid another patch, or maybe by that time Benno needs a v2 anyway.